### PR TITLE
Document the macOS unverified developer workaround

### PR DESCRIPTION
**Pull request by kingthorin**
Re-implement #498. Per that PR once this is accepted I'll also change core-help.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by thc202** [comment]
It's the getting started guide that needs to be updated (better done for 2.12).

**Comment by kingthorin** [comment]
Ping @psiinon could you do a minor update to that doc? (I'm only asking because I know you've dealt with them recently 😀 )

**Comment by psiinon** [comment]
Will do!

File: site/content/getting-started/index.md [hunk 1]
```diff
@@ -67,7 +67,9 @@ ZAP has installers for Windows, Linux, and Mac OS/X. There are also Docker image
 ##### Install ZAP
```
[the remark this answers is not in the document]
Comment (kingthorin) [reply site/content/getting-started/index.md:67]: Okay, I'll tackle that.

**Comment by kingthorin** [comment]
Tweaked

**Review by psiinon — APPROVED** [review]

File: site/content/getting-started/index.md [hunk 2]
```diff
@@ -62,12 +62,14 @@ Because ZAP is open-source, the source code can be examined to see exactly how t
...
 
 ##### Install ZAP
 The first thing to do is install ZAP on the system you intend to perform pentesting on. Download the appropriate installer from the [Download](/download/) page.
 
-Note that ZAP requires Java 8+ in order to run. The Mac OS/X installer includes an appropriate version of Java but you must install Java 8+ separately for Windows, Linux, and Cross-Platform versions. The Docker versions do not require you to install Java.
+Note that ZAP requires Java 11+ in order to run. The macOS installer includes an appropriate version of Java but you must install Java 11+ separately for Windows, Linux, and Cross-Platform versions. The Docker versions do not require you to install Java.
+
+Warning: OWASP ZAP is currently not a verified developer with Apple. On macOS, you will see a message like: `“OWASP ZAP.app” cannot be opened because the developer cannot be verified.` To circumvent this warning, you would need to go to "System Preferences" > "Security & Privacy" at the bottom of the dialog. You will see a message saying that "OWASP ZAP" was blocked. Next to it, if you trust the downloaded installer, you can click "Open anyway".
```
Comment (thc202) [inline site/content/getting-started/index.md:72]: There's no "Warning:" in the doc and the formatting is not exactly the same (e.g. missing macOS header, quoted error message). From the image in: https://github.com/zaproxy/zap-extensions/pull/4140#issuecomment-1280871607

Comment (thc202) [reply site/content/getting-started/index.md:72]: Should also be after the following sentence.

File: site/content/getting-started/index.md [hunk 3]
```diff
@@ -62,15 +62,21 @@ Because ZAP is open-source, the source code can be examined to see exactly how t
...
-Note that ZAP requires Java 8+ in order to run. The Mac OS/X installer includes an appropriate version of Java but you must install Java 8+ separately for Windows, Linux, and Cross-Platform versions. The Docker versions do not require you to install Java.
+Note that ZAP requires Java 11+ in order to run. The macOS installer includes an appropriate version of Java but you must install Java 11+ separately for Windows, Linux, and Cross-Platform versions. The Docker versions do not require you to install Java.
 
 Once the installation is complete, launch ZAP and read the license terms. Click **Agree** if you accept the terms, and ZAP will finish installing, then ZAP will automatically start.
 
+###### macOS
+OWASP ZAP is currently not a verified developer with Apple. On macOS, you will see a message like: 
+> “OWASP ZAP.app” cannot be opened because the developer cannot be verified.
+
+To circumvent this warning, you would need to go to "System Preferences" &gt; "Security & Privacy" at the bottom of the dialog. You will see a message saying that "OWASP ZAP" was blocked. Next to it, if you trust the downloaded installer, you can click "Open anyway".
```
Comment (thc202) [inline site/content/getting-started/index.md:78]:
```suggestion
To circumvent this warning, you would need to go to **System Preferences** &gt; **Security & Privacy** at the bottom of the dialog. You will see a message saying that "OWASP ZAP" was blocked. Next to it, if you trust the downloaded installer, you can click **Open anyway**.

**Comment by kingthorin** [comment]
Thanks, got that one now too.

**Review by thc202 — APPROVED** [review]

**Pull request merged**